pandunia-lekse.csv

```diff
@@ -2764,14 +2810,16 @@ wate||||watt||vatio|||||||||||||vato|watti|wat
 waw|||zho:哇 (wā), eng:wow|wow! (gee!)||guau (vaya)||вау!|||||||||||ŭaŭ!|oho (vau)|wow!, łoł!, jej!
 waxa|||eng:wash, deu:waschen, zho:洗 (xǐ)|wash||lavar||мыть|||||||||||lavi|pestä|prać, umyć, myć
 waxomaxin||||washing machine||lavadora||стиральная машина|||||||||||lavmaŝino|pesukone|pralka
-wayi||||outer (external)||exterior|||||||||||||ekstera|ulko- (ulkoinen)|zewnętrzny
-waya||||oust (expel)||expulsar||||||||||||||poistaa|
+waya||||be outside of (go outside)||estar fuera de|||||||||||||||
```

Comment:
me fikra -a mena fati (active) xey e -u mena beyi (passive) xey.

Reply:
si, a we no maca "nica", "bada", o "cirka". me fikra bizari sate da su ali dixe alo funca.

Reply:
me bixu fikra le...

Reply:
okey. me wil hafiza le sa yi sate.